app/waf/src/controllers/member.controller.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  post,
+  param,
+  get,
+  getFilterSchemaFor,
+  getWhereSchemaFor,
+  patch,
+  put,
+  del,
+  requestBody,
+} from '@loopback/rest';
+import {Member} from '../models';
+import {MemberRepository} from '../repositories';
+import uuid = require('uuid');
+
+const prefix = '/adcaas/v1';
+
+export class MemberController {
+  constructor(
+    @repository(MemberRepository)
+    public memberRepository: MemberRepository,
+  ) {}
+
+  @post(prefix + '/members', {
+    responses: {
+      '200': {
+        description: 'Member model instance',
+        content: {'application/json': {schema: {'x-ts-type': Member}}},
+      },
+    },
+  })
+  async create(@requestBody() member: Member): Promise<Member> {
+    member.id = uuid();
+    return await this.memberRepository.create(member);
```

Comment:
This would raise 500 internal server error, however, in many situation, it's not errors from server. but input error. 
It's ok for now. but should be prechecked in future.